lib/main.dart

```diff
@@ -22,12 +22,13 @@ class Spoonacular extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return GetMaterialApp(
+      getPages: Routes.routes,
+      initialRoute: TextConstants.homeRoute,
```

Comment:
Initial page need to be the splash i think

Reply:
Yes, I forgot to change it. Initial route is now the splash screen